src/components/snackbar/Snackbar.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useRef, useState } from 'react';
+
+import { useColorMode } from '@chakra-ui/react';
+
+import { closeDark, closeWhite } from '../../assets';
+
+import './snackbar.css';
+
+interface SnackbarProps {
+  type?: 'action' | 'action-long';
+  open: boolean;
+  autoCloseTime?: number;
+  canBeClosed?: boolean;
+  children: React.ReactNode;
+  actionProps?: JSX.Element;
+  align?:
+    | 'top-left'
+    | 'top-right'
+    | 'top-center'
+    | 'bottom-left'
+    | 'bottom-right'
+    | 'bottom-center';
+  onclose: (args: boolean) => void;
+}
+
+function Snackbar({
+  type = undefined,
+  open,
+  autoCloseTime = 3000,
+  canBeClosed = true,
+  align = 'top-right',
+  children,
+  actionProps,
+  onclose,
+}: SnackbarProps) {
+  const timerRef = useRef<ReturnType<typeof setTimeout> | null>(null);
+  const [isOpen, setisopen] = useState<boolean>(open);
```

Comment:
Please remove this redundant state. We are already getting the open state of snackbar from the parent. So we don't need to have seperate state from this.

Reply:
Addressed the comment.